ui/src/app/pages/login/account-creation/account-creation-survey.spec.tsx

```diff
@@ -5,9 +5,9 @@ import {
   AccountCreationSurvey,
   AccountCreationSurveyProps,
 } from 'app/pages/login/account-creation/account-creation-survey';
-import {getEmptyProfile} from 'app/pages/login/test-utils';
```

Comment:
deleted test-utils.  It had only this, which was a test-only copy of the real fn createEmptyProfile

Reply:
delete `createEmptyProfile()` ?

Hmm.  It's a bit of a code smell to me, but it's not one I have looked into much yet.  Probably worth doing so at some point.

Reply:
No, sorry, `getEmptyProfile

Reply:
No, sorry, `getEmptyProfile`. Though it looks like you may have gotten rid of the entire file. So. NVM